modules/samples/artifacts/PublishHttpOAuthRequest/PublishHttpOAuthRequestWithoutAccessToken.siddhi

```diff
@@ -0,0 +1,73 @@
+@App:name("PublishHttpOAuthRequest")
```

Comment:
Wrong siddhi app name here!!! This siddhi app will not get deployed. Have you tested this?